src/SystemView.h

```diff
@@ -363,6 +376,7 @@ class SystemMapViewport {
 	bool m_showGravpoints;
 	ShowLagrange m_showL4L5;
 	ShipDrawing m_shipDrawing;
+	CloudDrawing m_cloudDrawing;
```

Comment:
Note: I'm not really happy with this being in the Viewport class, as the Viewport is intended to be the low-level display code with no (runtime or compile-time) dependencies on the game context. I left the ship drawing state in this class during the last refactor to avoid the pain of rewriting `SetVisibility` into a proper API, but I'd like the "game" state handling to be moved into `SystemView` at some point.

This is not a concern for this PR at current, but is something I'd like to have done sooner or later.